pkgs/by-name/av/avdump3/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, writeScriptBin, fetchzip, dotnet-runtime, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "avdump3";
+  version = "8293_stable";
+
+  src = fetchzip {
+    url = "https://cdn.anidb.net/client/avdump3/avdump3_8293_stable.zip";
+    hash = "sha256-H9Sn3I4S9CmymKIMHVagDy+7svHs285S3EJgYQo+ks0=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/avdump3 $out/bin
+    mv * $out/share/avdump3
+    cat > $out/bin/avdump3 <<EOF
+    #!/bin/sh
+    export LD_LIBRARY_PATH="${lib.makeLibraryPath [ zlib ]}:\$LD_LIBRARY_PATH"
+    exec ${dotnet-runtime}/bin/dotnet $out/share/avdump3/AVDump3CL.dll "\$@"
+    EOF
+    chmod +x $out/bin/avdump3
+  '';
+
+  dontPatchELF = true;
+
+  meta = {
+    mainProgram = "avdump3";
+    description = "Tool for extracting audio/video metadata from media files and uploading it to AniDB";
```

Comment:
```suggestion
    description = "Extracting audio/video metadata from media files and uploading it to AniDB";
```

Reply:
Hmm, I don't think I agree with this one. Making the description a gerund (actually two conjoined gerunds, "extracting" and "uploading") sounds awkward to me. Plenty of packages' descriptions follow the pattern of "tool for", "program for", "app for", etc., even looking only at the relatively recent packages in `pkgs/by-name/`.

Reply:
Suggesting an opposite change for this:

```diff
-description = "Tool for extracting audio/video metadata from media files and uploading it to AniDB";
+description = "A tool for extracting audio/video metadata from media files and uploading it to AniDB";
```

Reply:
At some point there were some style guidelines added to the READMEs: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes

According to those guidelines, the description shouldn't start with an article like "A" or "The". On the other hand the guidelines also say that the description should be a "sentence" but that would require a verb -- doesn't make much sense to me :)

Reply:
What you have now should be fine